scripts/pomodoro.sh

```diff
@@ -13,6 +13,7 @@ pomodoro_auto_restart="@pomodoro_auto_restart"
 pomodoro_on="@pomodoro_on"
 pomodoro_complete="@pomodoro_complete"
 pomodoro_notifications="@pomodoro_notifications"
+pomodoro_granularity_seconds="@pomodoro_granularity_seconds"
```

Comment:
I think we could make this concise and just call it `pomodoro_granularity` and let the readme explain what it is